### PR TITLE
Creating contribution guide

### PR DESCRIPTION
**Pull request by maxim-engel-sap**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Contributing
+
+We are a small team and this project is one of many projects. We would love to see, when you help us developing new checks. Here a small how-to.
```
Comment (estevao-schultz-neto-SAP) [inline CONTRIBUTING.md:3]: We are a team with limited capacity dedicated to several projects. Every single contribution is valid and more than welcome. Feel free to help us in developing new ABAP Checks and keeping this project up & running! Hereby a short how-to tutorial.

Comment (maxim-engel-sap) [reply CONTRIBUTING.md]: Thanks, I changed that paragraph.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+    https://github.com/your-username/code-pal-for-abap.git
+
+Create and test your new Check in your system.
+
+When you are done. Please `stage` --> `commit` --> `push` the files to your fork.
+
+Back to github.com you can create a pull request from your fork to our base repo.
+
+This is the point, where we'll look into your code and approve the merge. Please create the request to merge with our `dev` branch, because our work-flow is from `dev` to `test` to `master`.
```
Comment (estevao-schultz-neto-SAP) [inline CONTRIBUTING.md:17]: At this point of time, we will verify your code and authorize/approve the merge (if applicable). Please create the request to merge it with our `dev` branch (our release workflow starts from `dev`, then to `test` and finally it reaches the `master` branch.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+Thank you in advance for contributing and for sharing your ideas with the community!
```
Comment (estevao-schultz-neto-SAP) [inline CONTRIBUTING.md:19]: Thank you in advance for contributing and sharing your ideas within our community! We really appreciate this!

**Comment by maxim-engel-sap** [comment]
Thanks @estevao-schultz-neto-SAP, I've included all your feedback.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
+# Contributing
+
+We are a team with limited capacity dedicated to several projects. Every single contribution is valid and more than welcome. Feel free to help us in developing new ABAP Checks and keeping this project up & running! Hereby a short how-to tutorial.
+
+### How-to
+
+Fork our repository, then clone/pull the forked repo via ABAPgit into your system:
+
+    https://github.com/your-username/code-pal-for-abap.git
```
Comment (lucasborin) [inline CONTRIBUTING.md:9]: I would say here that the user has to choose the `dev` branch (by default, it came as master).

**Review by eugen-guenther-sap — APPROVED** [review]

**Pull request merged**